<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string>background-color: rgb(50, 50, 50);color: rgb(255, 255, 255);selection-background-color:rgb(140, 140, 140)</string>
  </property>
  <widget class="QWidget" name="mainLayout_2">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
    <property name="margin">
     <number>4</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="layoutImageView" stretch=""/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="layoutControls">
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>520</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_2"/>
    <addaction name="actionImport_Image_Sequence_2"/>
    <addaction name="actionImport_Directory_2"/>
    <addaction name="separator"/>
    <addaction name="actionSave_2"/>
    <addaction name="actionSave_As_2"/>
    <addaction name="actionExport_to_Final_Cut_Pro"/>
    <addaction name="actionExport_to_Premiere"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuFPS_2">
     <property name="title">
      <string>FPS</string>
     </property>
     <addaction name="action24"/>
     <addaction name="action25"/>
     <addaction name="action48"/>
     <addaction name="action30"/>
     <addaction name="action50"/>
     <addaction name="action60"/>
     <addaction name="actionCustom_2"/>
    </widget>
    <addaction name="menuFPS_2"/>
    <addaction name="actionSet_Recording_Countdown"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
  </widget>
  <action name="actionImport_Image_Sequence">
   <property name="text">
    <string>Import Image Sequence...</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="action24">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Film (24 fps)</string>
   </property>
  </action>
  <action name="action25">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>PAL (25 fps)</string>
   </property>
  </action>
  <action name="action30">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>NTSC (30 fps)</string>
   </property>
  </action>
  <action name="action48">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show (48 fps)</string>
   </property>
  </action>
  <action name="action50">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>PAL Field (50 fps)</string>
   </property>
  </action>
  <action name="action60">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>NTSC Field (60 fps)</string>
   </property>
  </action>
  <action name="actionCustom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Custom...</string>
   </property>
  </action>
  <action name="actionCustom_2">
   <property name="text">
    <string>Custom...</string>
   </property>
  </action>
  <action name="actionSet_Recording_Countdown">
   <property name="text">
    <string>Set Recording Countdown...</string>
   </property>
  </action>
  <action name="actionImport_Directory">
   <property name="text">
    <string>Import Directory...</string>
   </property>
  </action>
  <action name="actionOpen_2">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionImport_Image_Sequence_2">
   <property name="text">
    <string>Import Image Sequence...</string>
   </property>
  </action>
  <action name="actionImport_Directory_2">
   <property name="text">
    <string>Import Directory...</string>
   </property>
  </action>
  <action name="actionSave_2">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As_2">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionExport_to_Final_Cut_Pro">
   <property name="text">
    <string>Export to Final Cut Pro...</string>
   </property>
  </action>
  <action name="actionExport_to_Premiere">
   <property name="text">
    <string>Export to Premiere...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
